tests/reporting/test_reporting_windows.py

```diff
@@ -0,0 +1,485 @@
+from ethereum.tools import tester
+from datetime import timedelta
+from utils import longToHexString, stringToBytes, bytesToHexString
+from pytest import fixture, raises
+from ethereum.tools.tester import ABIContract, TransactionFailed
+from reporting_utils import proceedToDesignatedReporting, proceedToRound1Reporting, proceedToRound2Reporting, proceedToForking, finalizeForkingMarket, initializeReportingFixture
+
+
+def test_reporting_window_initialize(localFixture, chain, mockUniverse, mockReportingAttendanceToken, mockReputationToken):
+    timestamp = chain.head_state.timestamp
+    mockUniverse.setReportingPeriodDurationInSeconds(timestamp)
```

Comment:
whats this value here? Reads a little confusingly as its setting a duration to a timestamp

Reply:
I'm a bit confused what _reportingWindowId is suppose to be a date? It's uint256 is that in ticks?

I can set _reportingWindowId as a timestamp and have  
startTime = _reportingWindowId * universe.getReportingPeriodDurationInSeconds();

universe.getReportingPeriodDurationInSeconds() == Reporting.reportingDurationSeconds() + Reporting.reportingDisputeDurationSeconds();

Reply:
The `ReportingPeriodDuration` is the total amount of time a reporting window takes to get to complete, which currently includes 27 days or reporting and 3 days for disputing the result.

The `ReportingWindowID` is essentially an index of these windows. So ID `0` would be the first 30 day window and ID `1` would be the window after it which takes place between days 30 and 60.

Right now this is setting the actual reporting period duration in seconds (which is normally 30 * 24 * 60 * 60) to whatever the current time in seconds since epoch is at the start of this test, which seems wrong to me

Reply:
My inclination is that instead of using the timestamp throughout the test as a time delta like this we should just be using some small understandable number like `10`.